src/amo/pages/Feedback/styles.scss

```diff
@@ -0,0 +1,113 @@
+@import '~amo/css/styles';
+
+$font-size-aside: $font-size-xs;
+
+.Feedback {
+  @include page-padding;
+
+  @include respond-to(large) {
+    display: flex;
+    justify-content: flex-start;
+  }
+}
+
+.Feedback-form-messages .Notice,
+.Feedback--Card {
+  margin-bottom: $padding-page;
+
+  ul {
+    padding-left: 0;
```

Comment:
We cannot use `padding-left` because that'll break in RTL mode. It should be `@include padding-start(0)` I think.